lemur/tests/test_certificates.py

```diff
@@ -230,7 +233,8 @@ def test_certificate_valid_years(client, authority):
         'owner': 'jim@example.com',
         'authority': {'id': authority.id},
         'description': 'testtestest',
-        'validityYears': 1
+        'validityYears': 1,
+        'dns_provider': None,
```

Comment:
The rest of the values passed to the api are `snakeCase` why is dns provider using `under_score`?